debs/SPECS/4.4.0/wazuh-agent/debian/rules

```diff
@@ -143,11 +144,14 @@ override_dh_install:
 	sed -i "s:WAZUH_HOME_TMP:${INSTALLATION_DIR}:g" src/init/templates/wazuh-agent.service
 	cp src/init/templates/wazuh-agent.service ${TARGET_DIR}/etc/systemd/system/
 
+	# Add debug symbols
+	mkdir -p ${TARGET_DIR_SYMS}/${INSTALLATION_DIR}
+	mv ${TARGET_DIR}/${INSTALLATION_DIR}/.symbols ${TARGET_DIR_SYMS}/${INSTALLATION_DIR}/.symbols
```

Comment:
Hi @jnasselle 

I'm trying to understand this line. There is some possibility that you want to copy/move the debug symbols from `src/symbols `to the ` ${TARGET_DIR_SYMS}/${INSTALLATION_DIR}/.symbols`

On the manager side, for example, you implemented the copy of debug symbols after compiling and building the manager with the command:

`cp src/symbols/* ${TARGET_DIR_SYMS}/${INSTALLATION_DIR}/.symbols`

The last command makes more sense IMHO.

Reply:
@HanesSciarrone , my answers below

> I'm trying to understand this line. There is some possibility that you want to copy/move the debug symbols from src/symbols to the ${TARGET_DIR_SYMS}/${INSTALLATION_DIR}/.symbols

 ${TARGET_DIR_SYMS} directory will contain the root directory for debuginfo package, and inside it will be ${INSTALLATION_DIR} which is /var/ossec.

[L89](https://github.com/wazuh/wazuh-packages/pull/1442/files/3e22a3ac9569002b21fce8d981c1ed1d36ef24b1#diff-f64a6f29d06337692baf79f75d2c2ee4feb8ac408b1372c4a770decaf88fba0dL89) copy all /var/ossec into  ${TARGET_DIR}/${INSTALLATION_DIR}, included .symbols, and this is wrong because ${TARGET_DIR} will only contain what today is deployed in our packages (a.k.a binary packet).
Said that, ${TARGET_DIR}/${INSTALLATION_DIR}/.symbols folder need to be moved to ${TARGET_DIR_SYMS}/${INSTALLATION_DIR}/.symbols

> On the manager side, for example, you implemented the copy of debug symbols after compiling and building the manager with the command:
> 
> cp src/symbols/* ${TARGET_DIR_SYMS}/${INSTALLATION_DIR}/.symbols

You are right, both should be the same!, but in the way that agent is doing it